impl/src/main/resources/META-INF/resources/jakarta.faces/faces-uncompressed.js

```diff
@@ -798,152 +500,78 @@ if (!((faces && faces.specversion && faces.specversion >= 23000 ) &&
          * Note:  This code originally from Sarissa: http://dev.abiss.gr/sarissa
          * It has been modified to fit into the overall codebase
          */
-        var getText = function getText(oNode, deep) {
-            var Node = {ELEMENT_NODE: 1, ATTRIBUTE_NODE: 2, TEXT_NODE: 3, CDATA_SECTION_NODE: 4,
-                ENTITY_REFERENCE_NODE: 5,  ENTITY_NODE: 6, PROCESSING_INSTRUCTION_NODE: 7,
-                COMMENT_NODE: 8, DOCUMENT_NODE: 9, DOCUMENT_TYPE_NODE: 10,
-                DOCUMENT_FRAGMENT_NODE: 11, NOTATION_NODE: 12};
-
-            var s = "";
-            var nodes = oNode.childNodes;
-            for (var i = 0; i < nodes.length; i++) {
-                var node = nodes[i];
-                var nodeType = node.nodeType;
-                if (nodeType == Node.TEXT_NODE || nodeType == Node.CDATA_SECTION_NODE) {
+        const getText = function getText(oNode, deep) {
+            // TODO: in what this function differs from Node.textContent?
```

Comment:
Node.textContent was only available in IE9. The original jsf.js needed to support IE6 as well.

Reply:
👍👍